sdk/head_tracker.cc

```diff
@@ -22,6 +22,65 @@
 #include "util/vectorutils.h"
 
 namespace cardboard {
+namespace {
+
+void GetViewporOrientationRotation(
```

Comment:
Since this function returns not only a rotation (transformation) but two instead, consider renaming it to a more explicit name.